src/extensions/default/QuickView/main.js

```diff
@@ -474,7 +477,8 @@ define(function (require, exports, module) {
 
         // Determine whether or not this URL/path is likely to be an image.
         var parsed = PathUtils.parseUrl(tokenString);
-        var hasProtocol = parsed.protocol !== "";
+        // If the URL has a protocol, check if it's one of the supported protocols
+        var hasProtocol = parsed.protocol !== "" && validProtocols.indexOf(parsed.protocol.trim().toLowerCase()) !== -1;
```

Comment:
Yeah, that would have been a better name , but as this was an existing variable , I haven't renamed it. 
Secondly , having (parsed.protocol !== "") check will help in cutting down the array search in lot of cases like a relative path image url.